apps/_pomodoro_app/realtime/application/LinkGenerator.js

```diff
@@ -0,0 +1,24 @@
+class LinkGenerator {
```

Comment:
It's not Java. Better to create just `function generateLink()`. There is no need to use a class

Reply:
Fixed!